lib/src/screens/home/feedTab.dart

```diff
@@ -12,8 +21,111 @@ class FeedTab extends StatefulWidget {
 }
 
 class _FeedTabState extends State<FeedTab> {
+  static final DateFormat postDateFormatter = DateFormat('h:m a');
+
+  final List<FeedItem> feedItems = [
+    FeedItem(
+      name: 'Lecture 1',
+      owner: 'Calculus I - AAB',
+      body: 'Please do this IMMEDIATELY',
+      type: FeedItemType.Lecture,
+      itemId: '54t3t3',
+      postDate: DateTime.now().add(Duration(hours: 4)),
+      attachmentUrl:
+          'https://images.all-free-download.com/footage_preview/mp4/kitty_68.mp4',
+    ),
+    FeedItem(
+      name: 'Homework 5',
+      owner: 'English 101',
+      body: 'Never do this',
+      type: FeedItemType.Homework,
+      itemId: '54t3t3',
+      postDate: DateTime.now().add(Duration(hours: 2)),
+    ),
```

Comment:
Agreed. Maybe do that now but hardcode the `HomeworkItem`?

Reply:
I chose to go with this "FeedItem" structure so that all actions that occur for a section can be added to user's feed collections, and the specific "item" that's tied to the action doesn't have to be loaded in the feed page until it's clicked.

Reply:
Yeah, I get what you mean, but in the long run, we will need to make a get request to get the `FeedItem`, and then make another get request for the `HomeworkItem` when the user taps on it, in which case we might as well use a `HomeworkItem`.

Reply:
Good point, I think we can discuss more in the meeting but I imagine quite a long feed. We could probably only show the first 10 or so items and get others later, but that would mean a lot of data to view the page.